packages/engine/Source/Scene/GaussianSplat3DTileContent.js

```diff
@@ -80,6 +81,43 @@ function GaussianSplat3DTileContent(loader, tileset, tile, resource) {
   this._transformed = false;
 }
 
+/**
+ * Performs checks to ensure that the provided tileset has the Gaussian Splatting extensions.
+ *
+ * @param tileset The tileset to check for the extensions.
+ * @returns {boolean} Returns <code>true</code> if the necessary extensions are included in the tileset.
+ * @static
+ */
+GaussianSplat3DTileContent.tilesetHasGaussianSplattingExt = function (tileset) {
```

Comment:
This may be nit-picky, but "has extension" != "requires extension", right? Should this be called `tilesetRequiresGaussianSplattingExt`?